### DISCUSSION TITLE
is it possible to use this OFFLINE? Alternatives if it gets down?

### DISCUSSION THREAD

**Discussion by AIhasArrived** [rany2/edge-tts#169] [Q&A] [answered]
Hello, 
How reliable is this? Can we assume it work for LIFE? What happenes if the service is not longer available?
In that case, is it possible to obtain this service OFFLINE in local somehow?

Thanks for answers

**Comment by rany2** [comment, marked as answer]
>  How reliable is this? Can we assume it work for LIFE? What happenes if the service is not longer available?

it has been working fine since Microsoft Edge got this feature, they haven't seem put in any effort to block this yet

>  In that case, is it possible to obtain this service OFFLINE in local somehow?

no
